DEVELOPMENT.md

```diff
@@ -43,6 +44,78 @@ For example, if we want to include UI tests when we test JVM target, call this:
 ```
 Don't run any background tasks, click mouse, or press keys during the tests. Otherwise, they probably fail.
 
-#### Run samples
-- First follow the instruction here: [Building JVM bindings](#building-jvm-bindings)
-- `./gradlew :SkiaAwtSample:run`
+## Run samples
+* First follow the instruction here: [Building JVM bindings](#building-jvm-bindings)
+### samples/SkiaAwtSample
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Pskiko.version=0.0.0-SNAPSHOT
+```
+ - skiko.fps.longFrames.show=true
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Dskiko.fps.longFrames.show=true -Pskiko.version=0.0.0-SNAPSHOT
+```
+ - skiko.renderApi=SOFTWARE
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Dskiko.renderApi=SOFTWARE -Pskiko.version=0.0.0-SNAPSHOT
+```
+ - skiko.vsync.enabled=false
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Dskiko.vsync.enabled=false -Pskiko.version=0.0.0-SNAPSHOT
+```
+ - skiko.transparency=true
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Dskiko.transparency=true -Pskiko.version=0.0.0-SNAPSHOT
+```
+ - skiko.swing.interop=true
+```bash
+./gradlew :skiko:publishAllSkikoJvmRuntimeToMavenLocal &&\
+./gradlew :SkiaAwtSample:run -Dskiko.swing.interop=true -Pskiko.version=0.0.0-SNAPSHOT
+```
+
+### samples/SkiaJsSample with mavenLocal
+If page in browser will be empty, try to reload it.
+```bash
+./gradlew publishToMavenLocal -p skiko -Pskiko.native.enabled=true -Pskiko.wasm.enabled=true &&\
+./gradlew jsBrowserDevelopmentRun -p samples/SkiaJsSample -Pskiko.version=0.0.0-SNAPSHOT
+```
+
+### samples/SkiaJsSample with skikoCompositeBuild=true
+It means: includeBuild("skiko") with dependency substitution. It compiles faster.
```

Comment:
```suggestion
When you add `-DskikoCompositeBuild=true`, Skiko from [sources](https://github.com/JetBrains/skiko/tree/master/skiko) is used (without this property the maven artifact is used). Every time you change sources, they are automatically applied when you run `jsBrowserDevelopmentRun`.
```

`includeBuild` usually doesn't tell what this property does, even if people are familiar with the Compose Gradle build feature. Also, it is used not to build project faster, but to use skiko from sources, not as prepublished maven artifact.

It also means, that we don't need to call `publishSkikoWasmRuntimePublicationToMavenLocal` and add  `-Pskiko.version=0.0.0-SNAPSHOT`.

But see my other suggestion, if we do that way (use Composite build by default), we don't need this section at all.

Reply:
We still need to call `publishSkikoWasmRuntimePublicationToMavenLocal`.
It's a different jar, packaged by tricky script.